src/main/java/com/github/romankh3/image/comparison/ImageComparison.java

```diff
@@ -327,6 +336,19 @@ private void joinToRegion(int row, int col) {
         }
     }
 
+    /**
+     * Returns the list of rectangles that would be drawn as a diff image.
+     * If you submit two images that are the same barring the parts you want to mask you get a list of rectangles that can be used as said mask
+     *
+     * @return List of {@link Rectangle}
+     */
+    public List<Rectangle> createMask() {
+        if (isImageSizesNotEqual(image1, image2)) {
+            throw new SizeMissMatchException("Size miss match on images");
+        }
```

Comment:
Hello. I as saw, newest version has own implementation of the response. Instead of the RuntimeException must be ComparisonState. It has state Size mismatch. 

Please, provide new patch of updates based on it. 

Also an open issue the fact that mask can contain rectangles which would not be
All inside the excluding area. 
@romankh3, what do you think?

Reply:
I agree that `RuntimeException` is not appropriate. It must be removed.
Instead of it, I'd like you(@mkytolai) to use `ComparisonState` enum to send the response.

Reply:
The method is used to create a list of rectangles that will be used for exclusion.
If we return a ComparisonState, then we can't return the rectangles which is the wanted result.

We can change it to work with a ComparisonState, but then we have to implement another method to get the result, and the original method createMask won't actually do anything besides just compare the size of the two images.

In the case that the images have a size miss match, what is the expected response if an exception is inappropriate?

On the topic of excluded exclusion parts, I agree that something should be done about it. Either several un-merged rectangles or a solution with a matrix of excluded pixels. Not sure which one is better at the moment. The matrix based solution would allow for a greater granularity at the expense of ease of use with manual input. A command line tool should probably be implemented so a user can manually mask parts of a sample image and then create the exclusion matrix.

I suggest we push the discussion about excluded exclusion further down the line, and make a working solution based on rectangles without holes.

Reply:
I agree, that ComparisonState, in this case, is not a valid response for the method.

But today I understood that this method is a stub for a future, right? So this check for size mismatch is not valid. 
let's remove it from the method and make a stub from this method `createMask()`. 

Waiting for your answer, sorry for the delay answer, I was very busy.

Best regards,
Roman.

Reply:
Removed sizeCheck and the exception.

Regardless if the method is expanded in the future or if something entirely else is implemented, there should be some way to easily get masking rectangles from a sample image.